docs/usage.rst

```diff
@@ -131,7 +131,9 @@ If you'd like to configure this behavior, e.g. to retry on some ``50x`` response
     from pystac_client import Client
     from pystac_client.stac_api_io import StacApiIO
 
-    retry = Retry(total=5, backoff_factor=1, status_forcelist=[502, 503, 504])
+    retry = Retry(
+        total=5, backoff_factor=1, status_forcelist=[502, 503, 504], allowed_methods=False
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏I'd recommend including a `frozenset` with `POST` additional to DEFAULT_ALLOWED_METHODS as input to be more explicit. If not, `allowed_methods=None` as recommended by the documentation.